src/js/videojs-dash.js

```diff
@@ -116,8 +122,26 @@ class Html5DashJS {
       delete this.player.dash;
     }
   }
+
+  /**
+   * Registers a callback function to be called before the MediaPlayer is initialized.
+   * Does not register duplicates
+   *
+   * @param {Function} fn callback function to register
+   * @function registerInitializationHook
+   */
+  static registerInitializationHook(fn) {
+    const index = Html5DashJS.beforeInitializeHooks_.indexOf(fn);
+
+    if (index <= -1) {
```

Comment:
`indexOf` according to its spec doesn't return a value less than `-1` so you can just make this `!== -1`.